.github/workflows/rust.yml

```diff
@@ -0,0 +1,25 @@
+name: Rust
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Install Protoc
+      uses: arduino/setup-protoc@v1
```

Comment:
Do we still need this? I know we had a previous protoc dependency for `cargo-shuttle`, but that was removed.

Reply:
> Do we still need this? I know we had a previous protoc dependency for `cargo-shuttle`, but that was removed.

Turns out, we do. Checking the build in my fork, removing it makes the build fail with this log:
```
error: failed to run custom build command for `opentelemetry-proto v0.1.0`

Caused by:
  process didn't exit successfully: `/home/runner/work/shuttle-examples/shuttle-examples/serenity/hello-world/target/debug/build/opentelemetry-proto-4258d7c978c9f9ca/build-script-build` (exit status: 101)
  --- stdout
  cargo:rerun-if-changed=src/proto/opentelemetry-proto/opentelemetry/proto/common/v1/common.proto
  cargo:rerun-if-changed=src/proto/opentelemetry-proto/opentelemetry/proto/resource/v1/resource.proto
  cargo:rerun-if-changed=src/proto/opentelemetry-proto/opentelemetry/proto/trace/v1/trace.proto
  cargo:rerun-if-changed=src/proto/opentelemetry-proto/opentelemetry/proto/trace/v1/trace_config.proto
  cargo:rerun-if-changed=src/proto/opentelemetry-proto/opentelemetry/proto/collector/trace/v1/trace_service.proto
  cargo:rerun-if-changed=src/proto/opentelemetry-proto/opentelemetry/proto/metrics/v1/metrics.proto
  cargo:rerun-if-changed=src/proto/opentelemetry-proto/opentelemetry/proto/collector/metrics/v1/metrics_service.proto
  cargo:rerun-if-changed=src/proto/opentelemetry-proto/opentelemetry/proto/logs/v1/logs.proto
  cargo:rerun-if-changed=src/proto/opentelemetry-proto/opentelemetry/proto/collector/logs/v1/logs_service.proto
  cargo:rerun-if-changed=src/proto/opentelemetry-proto

  --- stderr
  thread 'main' panicked at 'Could not find `protoc` installation and this build crate cannot proceed without
      this knowledge. If `protoc` is installed and this crate had trouble finding
      it, you can set the `PROTOC` environment variable with the specific path to your
      installed `protoc` binary.If you're on debian, try `apt-get install protobuf-compiler` or download it from https://github.com/protocolbuffers/protobuf/releases

  For more information: https://docs.rs/prost-build/#sourcing-protoc
  ', /home/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/prost-build-0.11.9/src/lib.rs:1457:10
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
warning: build failed, waiting for other jobs to finish...
```

Reply:
Thanks for trying that @DitherWither

Reply:
Ah, yes, we shouldn't need that anymore. It looks like your branch is on 0.16, while this requirement was removed in 0.18